src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -599,9 +597,11 @@ void AutomatedRssDownloader::updateFieldsToolTips(bool regex)
               + "<li>" + tr("Whitespaces count as AND operators (all words, any order)") + "</li>"
               + "<li>" + tr("| is used as OR operator") + "</li></ul></p>"
               + "<p>" + tr("If word order is important use * instead of whitespace.") + "</p>";
-        ui->lineContains->setToolTip(tip);
-        ui->lineNotContains->setToolTip(tip);
     }
+
+    tip += "<p>" + tr("An expression with an empty ") + "<tt>|</tt>" + tr(" clause (e.g. ") + "<tt>" + tr("expr|") + "</tt>" + tr(")");
```

Comment:
FYI: there's an alternative you can use: 
`tr("An expression with an empty %1 clause (e.g. %2").arg("<tt>|</tt>").arg()`